src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -1365,6 +1510,38 @@ ov::npuw::LLMCompiledModel::LLMCompiledModel(const std::shared_ptr<ov::Model>& m
     refine_dynamic_props(npuw_llm_props, npudesc);
     m_cfg.update(any_copy(npuw_llm_props));
 
+    const bool optimize_cb4 = m_cfg.get<::intel_npu::NPUW_LLM_OPTIMIZE_FP8E4M3>();
```

Comment:
Let name it `fp8` anywhere if you don't mind: `optimize_fp8`

Reply:
possible